challenges/qr_code_component/style.css

```diff
@@ -1,3 +1,5 @@
+@import url('https://fonts.googleapis.com/css2?family=Outfit:wght@100;200;300;400;500;600;700;800;900&display=swap');
```

Comment:
`Outfit` font doesn't seem to be used anywhere.

Reply:
Corrected